rotkehlchen/api/rest.py

```diff
@@ -4413,3 +4417,44 @@ def reprocess_skipped_external_events(self) -> Response:
             result=_wrap_in_ok_result({'total': total, 'successfull': successfull}),
             status_code=HTTPStatus.OK,
         )
+
+    def export_history_events(
+            self,
+            filter_query: HistoryBaseEntryFilterQuery,
+            directory_path: Path,
+    ) -> Response:
+        dbevents = DBHistoryEvents(self.rotkehlchen.data.db)
+        with self.rotkehlchen.data.db.conn.read_ctx() as cursor:
+            history_events, _, _ = dbevents.get_history_events_and_limit_info(
+                cursor=cursor,
+                filter_query=filter_query,
+                has_premium=True,
+                entries_limit=None,
+            )
+
+        if len(history_events) == 0:
+            return api_response(
+                wrap_in_fail_result('No history processed in order to perform an export'),
+                status_code=HTTPStatus.CONFLICT,
+            )
```

Comment:
returning an empty list seems weird, the error occurs from a user exporting if he has no events (or no events matching the criteria) since the endpoint exports a {result: boolean} normally, it just  returns {result: None}

Reply:
ah with an empty list I meant creating a file with no events but maybe is better to leave it as an error